Autopilot/PathManager/Inc/cruisingState.hpp

```diff
@@ -4,6 +4,7 @@
 */
 
 #include "pathDatatypes.hpp"
```

Comment:
No need to include this anymore. All required structs are in `TelemPathInterface.hpp` now